packages/manager/apps/dedicated/client/app/networkSecurity/scrubbing-center/scrubbing-center.controller.js

```diff
@@ -0,0 +1,124 @@
+import { PERIODS, PERIOD_LIST, API_PATH } from './scrubbing-center.constant';
+
+export default class ScrubbingCenterController {
+  /* @ngInject */
+  constructor($http, $translate, Alerter, Apiv2Service, OvhHttp) {
```

Comment:
**nitpick:** `OvhHttp` is deprecated. Please use `$http`
```suggestion
  constructor($http, $translate, Alerter, Apiv2Service) {
```

Reply:
You forgot to remove them from the constructor